shared/libraries/opcuatms/opcuatms/include/opcuatms/converters/dict_conversion_utils.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022-2023 Blueberry d.o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <opcuatms/opcuatms.h>
+#include <opcuashared/opcuavariant.h>
+#include <property_object_ptr.h>
+
+BEGIN_NAMESPACE_OPENDAQ_OPCUA_TMS
+
+class DictConversionUtils
```

Comment:
I don't really like the name of this class, I am open for suggestions.

Reply:
If anything, this is PropertyObjectConversionUtils